misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/JBallerinaDebugServer.java

```diff
@@ -464,6 +472,19 @@ public void sendOutput(String output, String category) {
         }
     }
 
+    public Value evaluateExpression(String expression, ThreadReference threadReference) {
```

Comment:
How about using a name like `evaluateBreakpointCondition`, as this util function is specifically written of that purpose? Also, Shall we modify the method to return a `boolean` instead of returning the direct result (which is a JDI value instance)?

Reply:
Done